src/postgres/commands/registerPostgresCommands.ts

```diff
@@ -24,7 +24,7 @@ import { openPostgresFunction } from "./openPostgresFunction";
 
 export function registerPostgresCommands(): void {
     ext.postgresCodeLensProvider = new PostgresCodeLensProvider();
-    ext.context.subscriptions.push(languages.registerCodeLensProvider({ pattern: `{**/,}*${postgresBaseFileName}*${sqlFileExtension}` }, ext.postgresCodeLensProvider));
+    ext.context.subscriptions.push(languages.registerCodeLensProvider({ pattern: `{**/,}*${postgresBaseFileName}*${postgresFileExtension}` }, ext.postgresCodeLensProvider));
```

Comment:
I feel like it would be fine to use the old logic based solely on the file extension, since it's now unique to postgres